Source/ThirdParty/RuntimeGifLibrary/RuntimeGifLibrary.Build.cs

```diff
@@ -0,0 +1,38 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+using System.IO;
+using UnrealBuildTool;
+
+public class RuntimeGifLibrary : ModuleRules
+{
+	public RuntimeGifLibrary(ReadOnlyTargetRules Target) : base(Target)
+	{
+		Type = ModuleType.External;
+
+		if (Target.Platform == UnrealTargetPlatform.Win64 || Target.Platform == UnrealTargetPlatform.Android)
+		{
+			PublicDefinitions.Add("WITH_LIBNSGIF=1");
+
+			string IncPath = Path.Combine(ModuleDirectory, "include");
+			PublicSystemIncludePaths.Add(IncPath);
+
+			PublicIncludePaths.Add(Path.Combine(ModuleDirectory, "include"));
```

Comment:
Btw,  you don't need to do both `PublicSystemIncludePaths.Add` and  `PublicIncludePaths.Add`

Reply:
Resolved